auto2cmake.py

```diff
@@ -1064,8 +1064,12 @@ def generate_default_cmake(req_dir):
     sources += ")\n"
 
     r_cmake_file = open(req_dir + "/CMakeLists.txt", "w")
-    r_cmake_file.write("cmake_minimum_required(VERSION 2.8)\n")
+    r_cmake_file.write("cmake_minimum_required(VERSION 3.24)\n")
     r_cmake_file.write(sources)
+#    r_cmake_file.write(        VERSION 1.0.0.0)
```

Comment:
Are you sure this needs to be here?.

Reply:
Will remove the outcommented lines in the next version. But would be good to have that information. Didn't find a way yet to deal with the to be included whitespaces in my project and to gather the needed information. (I'm a beginner.)

Reply:
put the string in quotes, like: `r_cmake_file.write("        VERSION 1.0.0.0")`. This https://www.geeksforgeeks.org/writing-to-file-in-python/ should give a good introduction for file handling in python